ci/stable_driver.sh

```diff
@@ -56,11 +56,14 @@ mkdir -p $stableroot/$datestr
 cd $stableroot/$datestr
 git clone $repo_url
 cd GDASApp
-git checkout feature/stable-nightly
 
 # ==============================================================================
-# merge in develop
-git merge develop
+# run ecbuild to get the repos cloned
```

Comment:
Not a request for change but rather information.  What is the purpose of this section of the script?   `stable_driver.sh` executes `run_ci.sh`   Script `run_ci.sh` executes `build.sh`.   Won't `build.sh` clone and build the JEDI repos required by GDASApp?

Reply:
This clones the repos so that one can run the command on them to get the most recent commits. The ecbuild command is mostly equivalent here to the `./checkout.sh` command in the workflow. There are probably other ways to do this but this was the path of least resistance.

Reply:
Thank you for the explanation.   We're not just building GDASApp.  We're checking if GDASApp builds with the current head of repo branches required by GDASApp.